src/app/wallets/add-invoice-for-wallet.ts

```diff
@@ -208,6 +261,7 @@ const addInvoice = async ({
   const dealer = DealerPriceService()
   const walletInvoiceBuilder = WalletInvoiceBuilder({
     dealerBtcFromUsd: dealer.getSatsFromCentsForFutureBuy,
+    dealerUsdFromBtc: dealer.getCentsFromSatsForFutureBuy,
```

Comment:
**This is a critical addition here**

Is this the right dealer method to do the conversion without introducing new 1-cent attacks?